examples/react-todos-app/vite.config.ts

```diff
@@ -18,12 +26,16 @@ export default defineConfig({
   plugins: [react()],
 
   test: {
+    reporters: ['default'],
     globals: true,
     cache: {
       dir: '../../node_modules/.vitest',
     },
     coverage: {
       reporter: ['lcov', 'text'],
+      provider: 'v8',
+      reportsDirectory: '../../coverage/react-todos-app',
```

Comment:
I noticed this is also specified in the `package.json`. Which one wins?

Reply:
You mean `project.json`. They're both set to the same value now. Previously, it was only in `project.json`, but after update it stopped having an effect, coverage files were being created in project directory. Moving it to `vitest.config` worked, and I would like to keep that as a single source of truth. Unfortunately, our coverage plugin (specifically the `getNxCoveragePaths` helpers) expects it in the `project.json`, so I had to copy it there as well :grimacing: 

I want to get rid of it in `project.json`, but since it has to be fixed in our coverage plugin first, I'm gonna create a follow-up issue for that.

(Duplicate of https://github.com/code-pushup/cli/pull/532#discussion_r1511467591, BTW)

Reply:
Created #536